Branch-SDK/Branch-SDK/BNCXcode7Support.h

```diff
@@ -6,14 +6,16 @@
 //  Copyright © 2016 Branch Metrics. All rights reserved.
 //
 
+#if defined(__IPHONE_OS_VERSION_MIN_REQUIRED) && __IPHONE_OS_VERSION_MIN_REQUIRED < 70000
+#error iOS 7 is the minimum supported Branch iOS version.
```

Comment:
it'd be great if we could silently fail on iOS 6 instead of crashing from a class not found error. Can you implement that path?